packages/mobile/src/firebase/firebase.ts

```diff
@@ -213,44 +214,60 @@ export function appVersionDeprecationChannel() {
   })
 }
 
+/*
+We use firebase remote config to manage feature flags.
+https://firebase.google.com/docs/remote-config
+
+This also allows us to run AB tests.
+https://firebase.google.com/docs/ab-testing/abtest-config
+*/
 export function appRemoteFeatureFlagChannel() {
   if (!FIREBASE_ENABLED) {
     return null
   }
 
-  const errorCallback = (error: Error) => {
-    Logger.warn(TAG, error.toString())
-  }
-
   return eventChannel((emit: any) => {
-    const emitter = (snapshot: FirebaseDatabaseTypes.DataSnapshot) => {
-      const flags = snapshot.val()
-      Logger.debug(`Updated feature flags: ${JSON.stringify(flags)}`)
-      emit({
-        hideVerification: flags?.hideVerification ?? false,
-        showRaiseDailyLimitTarget: flags?.showRaiseDailyLimitTargetV2 ?? undefined,
-        celoEducationUri: flags?.celoEducationUri ?? null,
-        celoEuroEnabled: flags?.celoEuroEnabled ?? false,
-        shortVerificationCodesEnabled: flags?.shortVerificationCodesEnabled ?? false,
-        inviteRewardsEnabled: flags?.inviteRewardsEnabled ?? false,
-        inviteRewardCusd: flags?.inviteRewardCusd ?? 1,
-        inviteRewardWeeklyLimit: flags?.inviteRewardWeeklyLimit ?? 5,
-        walletConnectEnabled: flags?.walletConnectEnabled ?? false,
-        rewardsABTestThreshold:
-          flags?.rewardsABTestThreshold ?? '0xffffffffffffffffffffffffffffffffffffffff',
-        rewardsPercent: flags?.rewardsPercent ?? 5,
-        rewardsStartDate: flags?.rewardsStartDate ?? 1622505600000,
-        rewardsMax: flags?.rewardsMax ?? 1000,
-      })
+    const emitter = (fetchedRemotely: boolean) => {
+      if (fetchedRemotely) {
+        Logger.debug('Configs were retrieved from the backend and activated.')
+        const flags: FirebaseRemoteConfigTypes.ConfigValues = remoteConfig().getAll()
+        emit({
+          hideVerification: flags.hideVerification.asBoolean(),
```

Comment:
Is it possible for `hideVerification` flag to be `undefined`?

Reply:
Yes, it is but now I use the setDefaults method in order to keep this from happening. It also has the nice effect that we can reuse the same defaults in our initial state. I wanted to avoid having 2 separate defaults defined.